article_store/core/settings.py

```diff
@@ -0,0 +1,144 @@
+import os
+import configparser
+
+
+PROJECT_NAME = 'article_store'
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+CFG_PATH = os.path.join(BASE_DIR, '.')
+CFG_NAME = 'app.cfg'
+CONF = configparser.ConfigParser()
+CONF.read(os.path.join(CFG_PATH, CFG_NAME))
+
+DEFAULT_LOG_DIR = CONF.get('logging', 'path')
+
+SECRET_KEY = CONF.get('django', 'secret_key')
+
+DEBUG = CONF.getboolean('django', 'debug', fallback=False)
+
+CI = CONF.getboolean('django', 'ci', fallback=False)
+
+ALLOWED_HOSTS = CONF.get('django', 'allowed_hosts').split(',')
+
+INTERNAL_IPS = '127.0.0.1'
+
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'django_filters',
+    'debug_toolbar',
+    'rest_framework',
+    'rest_framework_xml',
+    'articles',
+]
+
+MIDDLEWARE = [
+    'django.middleware.security.SecurityMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.middleware.common.CommonMiddleware',
+    'django.middleware.csrf.CsrfViewMiddleware',
+    'debug_toolbar.middleware.DebugToolbarMiddleware',
+    'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.messages.middleware.MessageMiddleware',
+    'django.middleware.clickjacking.XFrameOptionsMiddleware',
```

Comment:
is this something related to the admin UI? Seems a API wouldn't need to worry about this?